annlite/container.py

```diff
@@ -5,7 +5,7 @@
 from docarray import Document, DocumentArray
 from loguru import logger
 
-if TYPE_CHECKING:
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
Just curious, what's the effect of marking comment `# pragma: no cover`? to ignore the test coverage? With this goal, the benefits cannot convince me to make this change.

Reply:
Yes, it tells `pytest` not to count this line into coverage. Since none of the types introduced here will be used. It increased a lot of coverage in this patch😂.

Reply:
It's a simple trick, I will remove them then.